web/modules/custom/joinup_core/joinup_core.drush.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Drush commands related to joinup.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function joinup_core_drush_command() {
+  $items = [];
+
+  $items['joinup-status-check'] = [
```

Comment:
Then I think it needs some polishing, because I have this:
```
francesco in ~/Projects/web/joinup/web <git:ISAICP-2509>
[09:33:05] ❯❯❯ drush joinup-status-check                                                                                                                      ✱
The following changes were detected in the entity type and field definitions.                                                                        [error]

<!-- THEME DEBUG -->
<!-- THEME HOOK: 'item_list' -->
<!-- BEGIN OUTPUT from 'core/themes/stable/templates/dataset/item-list.html.twig' -->
<h3>Rdf entity</h3><ul><li>The <em class="placeholder">Translation source</em> field needs to be installed.</li><li>The <em
class="placeholder">Translation outdated</em> field needs to be installed.</li><li>The <em class="placeholder">Translation author</em> field needs to
be installed.</li><li>The <em class="placeholder">Translation status</em> field needs to be installed.</li><li>The <em
class="placeholder">Translation created time</em> field needs to be installed.</li><li>The <em class="placeholder">Translation changed time</em>
field needs to be installed.</li></ul>
<!-- END OUTPUT from 'core/themes/stable/templates/dataset/item-list.html.twig' -->


The rebuild_access setting is enabled in settings.php. It is recommended to have this setting disabled unless you are performing a rebuild.          [error]
The trusted_host_patterns setting is not configured in settings.php. This can lead to security vulnerabilities. It is <strong>highly                 [error]
recommended</strong> that you configure this. See <a href="https://www.drupal.org/node/1992030">Protecting against HTTP HOST Header attacks</a> for
more information.
Virtuoso version meets minimum requirements.                                                                                                         [error]
```